### PR TITLE
Kocherginv hw3

### PR DESCRIPTION
**Pull request by KocherginV**
Adding Hacker Rank Homework, days 3 and 4.

### CODE REVIEW

**Comment by KocherginV** [comment]
Refer to the  ebbc77b commit, please. Accidentally commited twice. Sorry for inconvenience.

File: homeworks/src/vkochergin/module1/example/Homework3/HackerRankDay4.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package vkochergin.module1.example.Homework3;
+
+/**
+ * Created by Volodymyr Kochergin on 14.04.2017.
+ */
+public class HackerRankDay4 {
```
Comment (PBaranovskyi) [inline homeworks/src/vkochergin/module1/example/Homework3/HackerRankDay4.java:8]: Where is main method?

**Review by PBaranovskyi — CHANGES_REQUESTED** [review]

File: homeworks/src/vkochergin/module1/example/Homework3/HackerRankDay4.java (hunk 1 above)
Comment (KocherginV) [inline homeworks/src/vkochergin/module1/example/Homework3/HackerRankDay4.java]: https://github.com/PBaranovskyi/learningJava/pull/24/commits/6127fc2ef20d4c3672ae5c07dd86e79d69e9db4e commit with main method is added

**Review by KocherginV — COMMENTED** [review]
Commit https://github.com/PBaranovskyi/learningJava/pull/24/commits/6127fc2ef20d4c3672ae5c07dd86e79d69e9db4e with main() is added

**Review by PBaranovskyi — APPROVED** [review]

**Pull request merged**